transport_nantes/mobilito/urls.py

```diff
@@ -16,4 +16,6 @@
          name='session_summary'),
 
     path('ajax/create-event/', views.create_event, name='event_creation'),
+    path('ajax/get-geocoded-address/', views.MapsGeocodingView.as_view(),
```

Comment:
Kind of a funny function name: it's getting an address, not a specially geo-encoded address.